adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureRDD.scala

```diff
@@ -95,10 +102,117 @@ object FeatureRDD {
 
     FeatureRDD(rdd, sd)
   }
+
+  /**
+   * @param feature Feature to convert to GTF format.
+   * @return Returns this feature as a GTF line.
+   */
+  private[features] def toGtf(feature: Feature): String = {
+    def escape(entry: (Any, Any)): String = {
+      entry._1 + " \"" + entry._2 + "\""
+    }
+
+    val seqname = feature.getContigName
+    val source = Option(feature.getSource).getOrElse(".")
+    val featureType = Option(feature.getFeatureType).getOrElse(".")
+    val start = feature.getStart + 1 // GTF/GFF ranges are 1-based
+    val end = feature.getEnd // GTF/GFF ranges are closed
+    val score = Option(feature.getScore).getOrElse(".")
+    val strand = Features.asString(feature.getStrand)
+    val frame = Option(feature.getFrame).getOrElse(".")
+    val attributes = Features.gatherAttributes(feature).map(escape).mkString("; ")
+    List(seqname, source, featureType, start, end, score, strand, frame, attributes).mkString("\t")
+  }
+
+  /**
+   * @param feature Feature to write in IntervalList format.
+   * @return Feature as a one line interval list string.
+   */
+  private[rdd] def toInterval(feature: Feature): String = {
+    val sequenceName = feature.getContigName
+    val start = feature.getStart + 1 // IntervalList ranges are 1-based
+    val end = feature.getEnd // IntervalList ranges are closed
+    val strand = Features.asString(feature.getStrand, emptyUnknown = false)
+    val attributes = Features.gatherAttributes(feature)
```

Comment:
The javadoc for IntervalList (all the spec I'm aware of) describes the last column as "Interval name (an, ideally unique, name for the interval)".  I haven't seen many of these files in the wild, so I don't know if `|` delimited attributes in the Interval name column is a reasonable thing to do.

Thus I found the previous code somewhat unsavory and arbitrary (as some hard coded keys are shoved into dbxrefs), see https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala#L199.

I hadn't got to round tripping; I guess I was planning to shove the Interval name column into `Feature.name` field when reading IntervalList format and use `Features.nameOf` when writing.  And then find a different file for the unit test.